### DISCUSSION TITLE
Terminology: Markers vs Annotations

### DISCUSSION THREAD

**Discussion by amacneil** [foxglove/community#557] [Q&A]
**Description**
We support [ImageAnnotations](https://foxglove.dev/docs/studio/messages/image-annotations) messages, but when these show up in the Foxglove UI we call them `Markers`.

We should rename the UI to say `Annotations` if this is our chosen terminology.

<img width="736" alt="image" src="https://user-images.githubusercontent.com/637671/194675166-8238e7fd-d64c-4c19-a4f2-1193a3bb8e43.png">

**Comment by jtbandes** [comment]
For ROS users, they are ImageMarkers. We won’t be taking that support away — should we change the button to say markers or annotations based on the data source profile?
